<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetCompraPendentes</class>
 <widget class="QWidget" name="WidgetCompraPendentes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditBusca"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="TableView" name="table">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelPedidosCompra">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Produtos pendentes</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPedidosCompra_2">
        <property name="text">
         <string>Buscar:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Comprar avulso</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButtonComprarAvulso">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Comprar</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="ItemBox" name="itemBoxProduto"/>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxQuantAvulso">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxQuantAvulsoCaixas">
        <property name="maximum">
         <double>99999.990000000005239</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Produto</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Quant.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Caixas</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Filtros:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_35">
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButtonExcel">
        <property name="text">
         <string>Excel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBoxStatus">
        <property name="title">
         <string>Todos</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="9" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroEntregue">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Entregue</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroRecebimento">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Em recebimento</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroEmEntrega">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Em entrega</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroColeta">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Em coleta</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroFaturamento">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Em faturamento</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroEstoque">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Estoque</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroCompra">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Em compra</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroPendentes">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Pendente</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroIniciados">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Iniciado</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroEntregaAgend">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Entrega Agend.</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroQuebra">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Quebra</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QCheckBox" name="checkBoxFiltroSul">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>SUL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer_6">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>290</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pushButtonPDF">
        <property name="text">
         <string>PDF</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>pushButtonPDF</zorder>
     <zorder>groupBoxStatus</zorder>
     <zorder>pushButtonExcel</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>ItemBox</class>
   <extends>QLineEdit</extends>
   <header>src/itembox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
